modules/ftp-client/src/main/java/org/dcache/ftp/client/dc/AbstractDataChannel.java

```diff
@@ -122,7 +122,7 @@ public static void registerHandler(int transferMode,
 	}
 	*/
 
-        logger.debug("registering handler for class " + clazz.toString() + "; id = " + id);
+        logger.debug("registering handler for class {}; id = {}", clazz.toString(), id);
```

Comment:
Hi,

yes that is true but my task was to replace all string concatenation with the proper slf4j format only. Therefore I didn't change "clazz.toString" to "clazz" in this pull request. I think it's not such a good ideas to mix different tasks. What do you think?